pyproject.toml

```
@@ -0,0 +1,33 @@
+[project]
+name = "brax-jumpy"
+description = "Common backend for JAX or numpy."
+version = "0.0.2"
+authors = [
+    { name = "Brax Authors", email = "no-reply@google.com" }
+]
+readme = "README.md"
+license = { file="LICENSE" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+urls = {Homepage = "http://github.com/google/brax"}
+
+dependencies = [
+    "jax",
+    "jaxlib",
+    "numpy",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest",
+]
+
+[build-system]
+requires = ["hatchling"]
```

Since it doesn't match `brax_jumpy.py` https://hatch.pypa.io/latest/plugins/builder/wheel/#default-file-selection

```suggestion
[tool.hatch.build.targets.wheel]
only-include = ["jumpy.py"]

[build-system]
requires = ["hatchling>=1.4.0"]
```